### PR TITLE
Basic 1835 cjw

### PR DESCRIPTION
**Pull request by cjwest**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,4 +37,6 @@ vendor
 linky-clicky-all-the-things.sh
 uatd6
 */behat.local.yml
+/products/*/behat.local.yml
```
Comment (sherakama) [inline .gitignore]: We should just change this to one entry for all behat.local.yml files.

Replace the entries in this file with just one ignore.

File: includes/features/drupal.org/xml_sitemap/xml_sitemap.feature [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 
-  @api
+  @api @live
   Scenario: XML Sitemap
     Given I am logged in as a user with the "administrator" role
     And I run cron
     When I go to "/sitemap.xml"
-    Then I should get a "200" HTTP response
\ No newline at end of file
+    Then I should get a "200" HTTP response
```
Comment (sherakama) [inline includes/features/drupal.org/xml_sitemap/xml_sitemap.feature:9]: Two tests for the same thing with different results?

File: .gitignore [hunk 3]
```diff
@@ -37,4 +37,5 @@ vendor
 linky-clicky-all-the-things.sh
 uatd6
 */behat.local.yml
+/*/*/behat.local.yml
```
Comment (sherakama) [inline .gitignore]: simply 'behat.local.yml' will do

File: includes/features/SU-SWS/nobots/nobots.feature [hunk 4]
```diff
@@ -8,3 +8,9 @@ Feature: NoBots
 #   Then I should see "Affiliates" in the "First sidebar" region
 #   Then I should see "Affiliates" in the "Content Head" region
 #   Then I should see 5 or more ".views-field-field-s-affiliate-image" elements
+
+  @dev
+  Scenario: robots.txt disallows crawling
+    When I go to "robots.txt"
```
Comment (sherakama) [inline includes/features/SU-SWS/nobots/nobots.feature]: Nobots doesn't do anything with the robots file. It is a http header.

File: includes/features/drupal.org/User/adminstrator_role.feature [hunk 5]
```diff
@@ -3,9 +3,9 @@ Feature: module
   As administrator
   I want to verify permissions and access
 
-@api
-Scenario: User is logged in
-Given I am logged in as a user with the "administrator" role
-And I am on "/user"
-Then I should see "History"
+  @safe @api
```
Comment (sherakama) [inline includes/features/drupal.org/User/adminstrator_role.feature]: Can we make this directory lowercase please? user vs User

File: includes/features/drupal.org/context/context.feature [hunk 6]
```diff
@@ -3,7 +3,7 @@ Feature: module
   As administrator
   I want to verify functionality of the context module
 
-  @api @javascript @destructive @dev
+  @dev @api @javascript @destructive
   Scenario: Enable Context HTTP Header
     Given the "context_http_header" module is enabled
```
Comment (sherakama) [inline includes/features/drupal.org/context/context.feature]: No.. we don't want this here.

File: includes/features/drupal.org/xml_sitemap/xml_sitemap.feature (hunk 2 above)
Comment (cjwest) [inline includes/features/drupal.org/xml_sitemap/xml_sitemap.feature:9]: Yup. One for when the site is in development @dev, and one is when the site is live @live.

File: includes/features/SU-SWS/nobots/nobots.feature (hunk 4 above)
Comment (cjwest) [inline includes/features/SU-SWS/nobots/nobots.feature]: fine.

File: includes/features/drupal.org/User/adminstrator_role.feature (hunk 5 above)
Comment (cjwest) [inline includes/features/drupal.org/User/adminstrator_role.feature]: weird. it's lowercase on my machine...

**Pull request merged**